scripts/check-theme-lark-imports.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+'use strict';
+
+/* eslint-env node */
+
+const chalk = require( 'chalk' );
+const path = require( 'path' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+
+const THEME_LARK_DIR_PATH = path.resolve( __dirname, '..', 'packages', 'ckeditor5-theme-lark', 'theme' );
+const REGEX_FOR_INDEX_IMPORTS = /(?<=@import ".\/)(.*)(?=";)/gm;
+const REGEX_FOR_MATCHING_COMMENTS = /\/\*(?:(?!\*\/).|\n)*\*\//gm;
+
+// Exit process when 'theme-lark' package does not exists.
+if ( !fs.existsSync( THEME_LARK_DIR_PATH ) ) {
+	process.exit( 0 );
+}
+
+// List of imported `CSS` files but not from main `index.css`.
+const listOfImportsFoundInSubfolders = [];
+
+// List of ignored paths or path templates.
+// If some of `CSS` file should be present in `theme` folder but it shouldn't be imported in main `index.css` please put it into this list.
+const ignoreList = [
+	'**/build/**',
+	'**/dist/**',
+	'**/node_modules/**',
+	'**/mixins/**',
+	'theme.css',
+	'index.css'
+];
+
+const globOptions = { cwd: THEME_LARK_DIR_PATH, ignore: ignoreList };
+
+// List of all paths to `CSS` files in `theme` folder of `theme-lark` package.
+const cssFilesPathsList = glob.sync( '**/*.css', globOptions );
+
+cssFilesPathsList.forEach( filePath => {
+	const fileContent = fs.readFileSync( path.join( THEME_LARK_DIR_PATH, filePath ), 'utf-8' );
+
+	// Remove all comments (included commented code).
+	const fileContentWithoutComments = fileContent.replaceAll( REGEX_FOR_MATCHING_COMMENTS, '' );
+
+	// Check if files other than `index.css` has `@import`.
+	const matchList = [ ...fileContentWithoutComments.matchAll( REGEX_FOR_INDEX_IMPORTS ) ];
+	const matchSimplifiedList = matchList.map( item => item[ 0 ] );
+
+	if ( !matchSimplifiedList.length ) {
+		return;
+	}
+
+	// Add paths to already imported files so we can exclude them from not imported ones.
+	matchSimplifiedList.forEach( item => {
+		listOfImportsFoundInSubfolders.push( path.dirname( filePath ) + '/' + item );
```

Comment:
To consider: normalizing paths using the `upath` package.

Reply:
Adressed in a1fb550.

Reply:
I've used `path` not `upath` because `normalize` function doesn't work properly on `Windows` :shrug: 

![](https://github.com/ckeditor/ckeditor5/assets/7074833/59dac2bb-9ea8-4b3a-994d-ee733f8c54f6)

Reply:
Please, take a look at https://www.notion.so/Use-upath-instead-of-path-80c87e7e4d354742a10736085493fa77.

In the past, we (maintainers of dev-tools) decided to use `upath` instead of `path`.

_I know this link does not work for viewers outside the organization, I am sorry._

Reply:
Adressed in ef23503.

After normalisation paths gathered by `globSync` all went well :tada: